Hi, it looks like there is a genuine issue in QC.
```
## sw/BUILD/QualityControl-latest/log
--
FAILED: Modules/GLO/CMakeFiles/O2QcGLO.dir/src/ITSTPCMatchingTask.cxx.o 
/sw/slc8_x86-64/GCC-Toolchain/v10.2.0-alice2-1/bin/c++ -DBOOST_ALL_NO_LIB -DBOOST_CONTAINER_DYN_LINK -DBOOST_ERROR_CODE_HEADER_ONLY -DBOOST_FILESYSTEM_DYN_LINK -DBOOST_IOSTREAMS_DYN_LINK -DBOOST_PROGRAM_OPTIONS_DYN_LINK -DBOOST_REGEX_DYN_LINK -DBOOST_SERIALIZATION_DYN_LINK -DDPL_ENABLE_BACKTRACE -DFAIRMQ_HAS_STD_FILESYSTEM=1 -DFMT_LOCALE -DO2QcGLO_EXPORTS -D__CLANG_SUPPORT_DYN_ANNOTATION__ -I/sw/SOURCES/QualityControl/v1.34.1/v1.34.1/Modules/GLO/include -I/sw/SOURCES/QualityControl/v1.34.1/v1.34.1/Modules/GLO/src -I/sw/SOURCES/QualityControl/v1.34.1/v1.34.1/Modules/GLO -I/sw/SOURCES/QualityControl/v1.34.1/v1.34.1/Framework/include -isystem /sw/slc8_x86-64/boost/v1.75.0-local1/include -isystem /sw/slc8_x86-64/FairLogger/v1.10.1-local1/include -isystem /sw/slc8_x86-64/fmt/7.1.0-local1/include -isystem /sw/slc8_x86-64/FairMQ/v1.4.42-local2/include/fairmq -isystem /sw/slc8_x86-64/FairMQ/v1.4.42-local2/include -isystem /sw/slc8_x86-64/ROOT/v6-24-06-local1/include -isystem /sw/slc8_x86-64/Common-O2/v1.6.0-local1/include -isystem /sw/slc8_x86-64/libInfoLogger/v2.2.0-local1/include -isystem /sw/slc8_x86-64/Monitoring/v3.8.10-local1/include -isystem /sw/slc8_x86-64/Configuration/v2.6.3-local1/include -isystem /sw/slc8_x86-64/Control-OCCPlugin/v0.27.0-local2/include/Occ -isystem /sw/slc8_x86-64/grpc/v1.34.0-alice2-local1/include -isystem /sw/slc8_x86-64/protobuf/v3.15.8-local1/include -isystem /sw/slc8_x86-64/zlib/v1.2.8-5/include -isystem /sw/slc8_x86-64/c-ares/v1.17.1-local1/include -isystem /sw/slc8_x86-64/re2/2019-09-01-local1/include -isystem /sw/slc8_x86-64/abseil/20200225.2-local1/include -isystem /sw/slc8_x86-64/O2/7321-slc8_x86-64-local1/include -isystem /sw/slc8_x86-64/O2/7321-slc8_x86-64-local1/include/GPU -isystem /sw/slc8_x86-64/FairRoot/v18.4.2-local2/include -isystem /sw/slc8_x86-64/arrow/v5.0.0-alice2-local1/include -isystem /sw/slc8_x86-64/Vc/1.4.1-local1/include -isystem /sw/slc8_x86-64/ms_gsl/3.1.0-local1/include -isystem /sw/slc8_x86-64/libjalienO2/0.1.3-local1/include -isystem /sw/slc8_x86-64/RapidJSON/v1.1.0-alice2-local1/include -isystem /sw/slc8_x86-64/libuv/v1.40.0-local1/include -fPIC -O2 -std=c++17 -Wno-error=deprecated-declarations -Wno-error=unused-function -O2 -g -DNDEBUG -fPIC -std=gnu++17 -MD -MT Modules/GLO/CMakeFiles/O2QcGLO.dir/src/ITSTPCMatchingTask.cxx.o -MF Modules/GLO/CMakeFiles/O2QcGLO.dir/src/ITSTPCMatchingTask.cxx.o.d -o Modules/GLO/CMakeFiles/O2QcGLO.dir/src/ITSTPCMatchingTask.cxx.o -c /sw/SOURCES/QualityControl/v1.34.1/v1.34.1/Modules/GLO/src/ITSTPCMatchingTask.cxx
/sw/SOURCES/QualityControl/v1.34.1/v1.34.1/Modules/GLO/src/ITSTPCMatchingTask.cxx: In member function 'virtual void o2::quality_control_modules::glo::ITSTPCMatchingTask::initialize(o2::framework::InitContext&)':
/sw/SOURCES/QualityControl/v1.34.1/v1.34.1/Modules/GLO/src/ITSTPCMatchingTask.cxx:74:55: error: 'class o2::globaltracking::MatchITSTPCQC' has no member named 'getHistoFractionITSTPCmatch'; did you mean 'getFractionITSTPCmatch'?
   74 |   getObjectsManager()->startPublishing(mMatchITSTPCQC.getHistoFractionITSTPCmatch());
      |                                                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                       getFractionITSTPCmatch
```

--- a/Detectors/GlobalTracking/include/GlobalTracking/MatchITSTPCQC.h
+++ b/Detectors/GlobalTracking/include/GlobalTracking/MatchITSTPCQC.h
@@ -43,6 +43,7 @@ class MatchITSTPCQC
   void reset();
   TH1F* getHistoPtTPC() const { return mPtTPC; }
   TEfficiency* getFractionITSTPCmatch() const { return mFractionITSTPCmatch; }
+  TEfficiency* getHistoFractionITSTPCmatch() const { return mFractionITSTPCmatch; }
   TH1F* getHistoPt() const { return mPt; }
   TH1F* getHistoEta() const { return mEta; }
   TH1F* getHistoChi2Matching() const { return mChi2Matching; }

--- a/Detectors/GlobalTracking/src/MatchITSTPCQC.cxx
+++ b/Detectors/GlobalTracking/src/MatchITSTPCQC.cxx
@@ -112,7 +112,7 @@ void MatchITSTPCQC::run(o2::framework::ProcessingContext& ctx)
       mChi2Matching->Fill(trk.getChi2Match());
       mChi2Refit->Fill(trk.getChi2Refit());
       mTimeResVsPt->Fill(trkTpc.getPt(), trk.getTimeMUS().getTimeStampError());
-      LOG(INFO) << "*** chi2Matching = " << trk.getChi2Match() << ", chi2refit = " << trk.getChi2Refit() << ", timeResolution = " << trk.getTimeMUS().getTimeStampError();
+      LOG(DEBUG) << "*** chi2Matching = " << trk.getChi2Match() << ", chi2refit = " << trk.getChi2Refit() << ", timeResolution = " << trk.getTimeMUS().getTimeStampError();
       ++mNITSTPCSelectedTracks;
     }
   }

